content/v/tauron-arena.md

```diff
@@ -7,10 +7,12 @@ authors = ["M3n747"]
 +++
 
 Tauron Arena is a multipurpose arena located in Kraków, used for sporting events, concerts, religious rallies etc. The arena has the maximum capacity of 22&nbsp;000 people, making it the largest such venue in Poland.
-
-Since 2022, [Prime Time Wrestling](@/o/ptw.md) have been holding annual shows at Ryucon, a manga and anime-themed convention taking place at the venue. In 2025, however, [Legacy of Wrestling](@/o/low.md) was booked to hold their [third show](@/e/low/2025-07-11-low-3.md) at Ryucon instead.
+The arena also includes an adjacent, smaller hall (dubbed "Small Hall"), which can be used for sports as well as for conferences and expositions.
+Since 2022, [Prime Time Wrestling](@/o/ptw.md) have been holding annual shows at Ryucon, a manga and anime-themed convention taking place at the venue. First Ryucon show took place in a Small Hall, while 2023 and 2024 took place in the main venue.
```

Comment:
```suggestion
Since 2022, [Prime Time Wrestling](@/o/ptw.md) have been holding annual shows at Ryucon, a manga and anime-themed convention taking place at the venue. The first Ryucon show took place in the Small Hall, while 2023 and 2024 editions moved to the main arena.
```